incubator/k8s-spot-rescheduler/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "k8s-spot-rescheduler.fullname" . }}
+  labels:
+    app: {{ template "k8s-spot-rescheduler.name" . }}
+    chart: {{ template "k8s-spot-rescheduler.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "k8s-spot-rescheduler.fullname" . }}
+subjects:
+- kind: ServiceAccount
```

Comment:
Please choose a consistent list indentation style. It's mostly two spaces, so please apply this everywhere. Don't mix styles.

Reply:
Hm, not sure where I mixed styles? I used 2 spaces everywhere.

Reply:
Ahh, I see. Will fix.